### PR TITLE
Fix group roles mapping permissions

### PR DESCRIPTION
**Pull request by daniel-pimenta-DME**
Enables Role mapping tab when user has `manage` permission for the group

Closes issue #20125

### CODE REVIEW

**Review by jonkoops — CHANGES_REQUESTED** [review]
Please ensure there are no unnecessary code formatting changes and follow the configuration of the linter.

File: js/apps/admin-ui/src/components/role-mapping/AddRoleMappingModal.tsx [hunk 1]
```diff
@@ -32,17 +33,20 @@ type AddRoleMappingModalProps = {
...
-  onClose,
-}: AddRoleMappingModalProps) => {
+                                      id,
+                                      name,
+                                      type,
+                                      isRadio = false,
+                                      isLDAPmapper,
+                                      onAssign,
+                                      onClose,
+                                    }: AddRoleMappingModalProps) => {
```
Comment (ssilvert) [inline js/apps/admin-ui/src/components/role-mapping/AddRoleMappingModal.tsx:43]: Looks like these lines didn't change?  But introduced some spaces?

**Review by ssilvert — CHANGES_REQUESTED** [review]

**Comment by ssilvert** [comment]
@daniel-pimenta-DME Please fix conflicts.

**Comment by ssilvert** [comment]
@daniel-pimenta-DME I'm going to go ahead and close this.  We can reopen later if you want to pick it up.

**Pull request closed without merging**